File: FE/src/types/comic-chapter.ts
export interface CreateComicChapterRequest {
	comic_id: number;
	chapter: number;
	content: string;
}

export interface UpdateComicChapterRequest {
	id: number;
	comic_id: number;
	chapter: number;
	content: string;
}

export interface ComicChapterResponse {
	id: number;
	comic_id: number;
	chapter: number;
	content: string;
	created_at: string;
	updated_at: string;
}

File: BE/DTOs/Response/ComicChapter.cs
namespace TruyenCV.DTOs.Response;

public class ComicChapterResponse
{
	public string id { get; set; }
	public string comic_id { get; set; }
	public int chapter { get; set; }
	public string content { get; set; }
	public int key_require { get; set; }
	public DateTime? key_require_until { get; set; }
	public DateTime created_at { get; set; }
	public DateTime updated_at { get; set; }
}
